src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -258,56 +274,76 @@ def load_arguments(self, _):
         c.argument('kubernetes_version',
                    completer=get_k8s_versions_completion_list)
         c.argument('dns_name_prefix', options_list=['--dns-name-prefix', '-p'])
-        c.argument('node_osdisk_diskencryptionset_id', options_list=['--node-osdisk-diskencryptionset-id', '-d'])
+        c.argument('node_osdisk_diskencryptionset_id', options_list=[
```

Comment:
Would be great if you could undo the format change (lines of less than 120 characters are allowed)